### PR TITLE
bug 83048 - don't render html

### PR DESCRIPTION
**Pull request by CathLass**

### CODE REVIEW

**Review by acarbutt — APPROVED** [review]

**Comment by acarbutt** [comment]
Nice work

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml [hunk 1]
```diff
@@ -1,21 +1,13 @@
 ﻿@model GeneralInformationViewModel
 @{
-	var kw = WidthOverride.TwoThirds;
-	var distance = "";
+		var kw = WidthOverride.TwoThirds;
```
Comment (aishajilani) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml:3]: Unsure how this looks in VS, but the indentation appears to have an extra tab. Please double check.

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
+		var distance = "";
```
Comment (aishajilani) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml:4]: Unsure how this looks in VS, but the indentation appears to have an extra tab. Please double check.

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml [hunk 3]
```diff
@@ -1,21 +1,13 @@
...
 
-	if (!string.IsNullOrEmpty(Model.DistanceFromSchoolToTrustHeadquarters))
-	{
-		distance = $"{Model.DistanceFromSchoolToTrustHeadquarters} miles";
-		if (!string.IsNullOrEmpty(Model.DistanceFromSchoolToTrustHeadquartersAdditionalInformation))
```
Comment (aishajilani) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml:9]: Please double check the indentation for the if{} statement code block.

**Review by aishajilani — APPROVED** [review]

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml (hunk 3 above)
Comment (CathLass) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml:9]: checked  - no extra tab present

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml (hunk 2 above)
Comment (CathLass) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Components/GeneralInformation/default.cshtml:4]: checked  - no extra tab present

**Pull request merged**